tests/v2/validation/rbac/globalrolesv2/README.md

```diff
@@ -0,0 +1,90 @@
+# Global Roles v2
+Global Roles v2 introduces enhanced capabilities, allowing users to define permissions across all downstream clusters. This update aims to address the limitations of predefined roles, particularly challenges associated with the Restricted Admin role. 
+
+## Pre-requisites
+- Ensure you have an existing cluster that the user has access to. If you do not have a downstream cluster in Rancher, create one first before running this test.
+- Some tests require creating additional downstream cluster. Providing the provisioningInput parameter with appropriate values is mandatory unless you are skipping those tests.
+
+## Test Setup
+Your GO suite should be set to `-run ^TestGlobalRolesV2TestSuite$`. You can find specific tests by checking the test file you plan to run.
+
+In your config file, set the following:
+```json
```

Comment:
This seems to be in YAML but is defined as JSON, can you clarify?

Reply:
Copy paste error :) Fixed it.